requirements.md

```diff
@@ -0,0 +1,117 @@
+# Requirements for Open Screen Protocol
+
+This document outlines functional and non-functional requirements of the Open
+Screen protocol.
+
+Functional requirements derive from the two Web APIs that the protocol will support:
+- The [Presentation API](https://w3c.github.io/presentation-api/)
+- The [Remote Playback API](https://w3c.github.io/remote-playback/)
+
+Non-functional requirements address efficiency, user experience, and security
+aspects of the protocols.
+
+## Presentation API
+
+The Presentation API allows one browser (the *controlling user agent*, or
+*controller*) to discover and initiate presentation of Web content on a second
+user agent (the *receiving user agent*, or *receiver*).  We use the term
+*display* to refer the entire device responsible for implementing the
+*receiver*, including browser, OS, networking and display.
+
+A *presentation* is initated at the request of a *controlling browsing context*
+(Web page), which creates a *receiving browsing context* to load a *presentation
+request URL* and exchange messages with the resulting presentation.
+
+For detailed information about the behavior of the Presentation API operations
+described below, please consult the Presentation API specification.
+
+### <a name="REQ-P1"></a>Presentation Display Availability
+
+1. A controller must be able to discover the presence of a display connected to
+the same IPv4 or IPv6 subnet and reachable by IP multicast.
+
+2. A controller must be able to obtain the IPv4 or IPv6 address of the display,
+a friendly name for the display, and an IP port number for establishing a
+network transport to the display.
+
+3. A controller must be able to determine if the receiver is capable of
+displaying a specific presentation request URL.
```

Comment:
Done - alignment with the spec here should be done as much as possible.  Thanks for checking.